scripts/bootstrapCatalogWithSam.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -lt 5 ]]; then
+    echo "usage $0 <sam-url> <auth-token> <resource-id> <admin-group-name> <your-email>"
+    exit 1
+fi
+
+RESOURCE_TYPE="catalog"
+SAM_URL=$1
+AUTH_TOKEN=$2
+# resource_id such as broad-tdc-dev, broad-tdc-perf etc.
+RESOURCE_ID=$3
+ADMIN_GROUP_NAME=$4
+YOUR_EMAIL=$5
+
+# Create the admin group for the env
+curl -X POST -H 'Content-Length: 0' -H "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${ADMIN_GROUP_NAME}"
+# Add the user as an admin of the ops group
+curl -X PUT -H 'Content-Length: 0' -H "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${ADMIN_GROUP_NAME}/admin/${YOUR_EMAIL}"
```

Comment:
These are using `-H 'Content-Length: 0'` but I don't think that's required. Do they work without this directive?